src/basis/template/html.js

```diff
@@ -348,6 +348,21 @@
    /**
     * @func
     */
+    var bind_attrNS = function(domRef, attrName, namespace, oldValue, newValue){
```

Comment:
`namespace` before `attrName` is better order, like in setAttributeNS/removeAttributeNS methods